tools/update-zuliprc-api-field

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+"""
+This tool is for updating API key field `zuliprc` files of dummy users
+in development server.
+Usage:
+./tools/update-zuliprc-api-field /path/to/zuliprc_dev
+"""
+
+import argparse
+import configparser
+import subprocess
+import os
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description = '''
+    Tool to update API key field of `zuliprc` file with the
+    correct keys from database. Ensure running this outside of
+    vagrant environment.
+    ''')
+    parser.add_argument('path', metavar='FILE', type=str, nargs='+',
+                        help='config file of dummy users in development server')
+    args = parser.parse_args()
+
+    zuliprc = configparser.ConfigParser()
+    zuliprc_paths_list = args.path
+
+    for zuliprc_path in zuliprc_paths_list:
+        try:
+            with open(zuliprc_path, 'r') as f:
+                zuliprc.read_file(f, zuliprc_path)
+
+            if zuliprc.has_section('api'):
+                api_details = zuliprc['api']
+                if all(item in zuliprc.options('api') for item in ['email', 'key', 'site']):
+                    email = api_details['email']
+                    key = api_details['key']
+                    site = api_details['site']
+                    if 'localhost' not in site:
+                        print('Script to be used for development server config files only')
+                else:
+                    print('Config file missing required fields')
+                    continue
+
+        except FileNotFoundError:
+            print('No zuliprc found at path %s' % (zuliprc_path,))
+            continue
+        except configparser.MissingSectionHeaderError:
+            print('Failed to parse zuliprc file at %s' % (zuliprc_path,))
+            continue
+
+        is_vagrant_env = '.vagrant' in os.listdir('.')
+        if is_vagrant_env:
+            arguments = ['vagrant', 'ssh', '--command',
+                         'python manage.py print_initial_password ' + email]
+        else:
+            # Support users who don't have vagrant based setup
+            arguments = ['python', 'manage.py',
+                         'print_initial_password', email]
```

Comment:
It might be cleaner to add an `is_vagrant_env_host()` function in `scripts/lib/zulip_tools.py` and call that from here, to help avoid future code duplication.